addons/scratchstats/userscript.js

```diff
@@ -0,0 +1,115 @@
+function createItem(number, label) {
+  const item = document.createElement("div");
+  const numberDiv = document.createElement("div");
+  item.appendChild(numberDiv);
+  numberDiv.className = "sa-stats-number";
+  numberDiv.innerText = number;
+  const labelDiv = document.createElement("div");
+  item.appendChild(labelDiv);
+  labelDiv.innerText = label;
+  return item;
+}
+
+export default async function ({ addon, msg, console }) {
+  const username = location.pathname.split("/")[2];
+  if (!username) return;
+  const content = document.querySelector("#content");
+  const commentBox = document.querySelector("#content > .box:not(#profile-data):not(.slider-carousel-container)");
+  const statsBox = document.createElement("div");
+  content.insertBefore(statsBox, commentBox);
+  statsBox.className = "box sa-stats slider-carousel-container";
+  const statsHeader = document.createElement("div");
+  statsBox.appendChild(statsHeader);
+  statsHeader.className = "box-head";
+  const statsTitle = document.createElement("h4");
+  statsHeader.appendChild(statsTitle);
+  statsTitle.innerText = msg("title");
+  const statsMoreLink = document.createElement("a");
+  statsHeader.appendChild(statsMoreLink);
+  statsMoreLink.innerText = msg("view-more");
+  statsMoreLink.href = "https://scratchstats.com/" + username;
```

Comment:
target _blank?

Reply:
I don't like `target="_blank"` because I'm often confused by it when I encounter it on other sites. I expect to be able to use the back button to get back after opening a link, unless I middle clicked it. See [this article](https://css-tricks.com/use-target_blank/) for a more detailed explanation.

I don't want to add it unless there is a reason other than it being an external link (e.g. similar links on Scratch itself, but I don't think those exist).